composer.json

```diff
@@ -1,23 +1,63 @@
 {
-  "name": "",
-  "description": "",
```

Comment:
for description , i propose this [ "description": "the backend part for the project ImNotFoolJustSick"